arm/Microsoft.Automation/automationAccounts/jobSchedules/deploy.bicep

```diff
@@ -0,0 +1,53 @@
+@description('Optional. Name of the Automation Account job schedule. Must be a GUID. If not provided, a GUID is generated leveraging runbookName and scheduleName.')
+param name string = ''
+
+@description('Required. Name of the parent Automation Account.')
+param parent string
```

Comment:
something we should probably align on. Again, it does not matter for this module, but in other modules with multiple levels of nesting this won't work (as we will have parent of parent etc.). We can keep it here as is, or use `automationAccountName` instead. Something to think about

Reply:
Agree. Replaced with `automationAccountName`

Reply:
check